### DISCUSSION TITLE
Segmentation violation executing trivy k8s with v0.50.0

### DISCUSSION THREAD

**Discussion by weisinc** [aquasecurity/trivy#6398] [Q&A]
### Question

I'm facing issue while running trivy k8s:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x72b0246]
```
Command I used include:
trivy k8s pods --scanners vuln --report all
trivy -d k8s cluster --report summary

None of them is working for me. Anyone else facing it ?

### Target

Kubernetes

### Scanner

Vulnerability

### Output Format

None

### Mode

Standalone

### Operating System

macOS Sonoma 14.4.1

### Version

```bash
% trivy version
Version: 0.50.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-03-26 18:10:27.317281487 +0000 UTC
  NextUpdate: 2024-03-27 00:10:27.317281116 +0000 UTC
  DownloadedAt: 2024-03-26 19:56:02.673055 +0000 UTC
```

**Comment by simar7** [comment]
Might be related https://github.com/aquasecurity/trivy/issues/6391

**Reply by mikaello** [reply]
Do you face the same error in as well [Trivy 0.49.1](https://github.com/aquasecurity/trivy/releases/tag/v0.49.1)? If not I think this is a duplicate of #6391.

**Reply by weisinc** [reply]
I ran the same trivy command with 0.50.1, and there is no more error. Appears to be duplicate of [#6391].
Thanks @simar7 , @mikaello ... will close this discussion.
